core/src/main/java/io/github/wysohn/triggerreactor/core/main/TriggerReactorCore.java

```diff
@@ -1053,6 +1053,21 @@ public void onSave(String script) {
                         }
 
                         getInvEditManager().startEdit((IPlayer) sender, trigger);
+                        return true;
+                    } else if (args.length > 3 && args[2].equalsIgnoreCase("settitle")) {
+                        String name = args[1];
+                        String title = mergeArguments(args, 3, args.length-1);
+
+                        TriggerInfo info = Optional.of(getInvManager())
+                                .map(man -> man.get(args[1]))
+                                .map(Trigger::getInfo)
+                                .orElseThrow(() -> new RuntimeException("Missing TriggerInfo"));
```

Comment:
Wouldn't this also throw when there is no inventory trigger named `args[1]`?

Users will see an awkward red message in such a case.

Please test this in-game yourself to make sure everything is working as intended!

Reply:
oops. I'll check it out next time!

I used another code above my code.

https://github.com/TriggerReactor/TriggerReactor/blob/82eb85fb176ea78c4d6cb8d1c01bffd9bea70555/core/src/main/java/io/github/wysohn/triggerreactor/core/main/TriggerReactorCore.java#L648

Reply:
Yes, but when `man.get(args[1])` returns `null` because there is no inventory with the name exist, what would happen?

Reply:
You may try the command yourself `trg i someInventoryThatDoesntExist settitle abc`

Reply:
Oh. I had to check if args[0] inventory trigger is exist